rust/kcl-lib/src/execution/mod.rs

```diff
@@ -1933,23 +1933,23 @@ notMember = !obj.a
 "#;
         assert_eq!(
             parse_execute(code4).await.unwrap_err().message(),
-            "Cannot apply unary operator ! to non-boolean value: number",
+            "Cannot apply unary operator ! to non-boolean value: number(default units)",
         );
 
         let code5 = "
 a = []
 notArray = !a";
         assert_eq!(
             parse_execute(code5).await.unwrap_err().message(),
-            "Cannot apply unary operator ! to non-boolean value: array (list)",
+            "Cannot apply unary operator ! to non-boolean value: [any; 0]",
```

Comment:
I think it would be clearer to users if it said

"Cannot apply unary operator ! to non-boolean value: Array[any; 0]"

WDYT?